packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-authjs/main/angular/sign-in-form.md

```diff
@@ -0,0 +1,76 @@
+Build a sign-in page that captures both the username and password. As an example, see the app component files such as `app.component.html` and `app.component.ts` that generate a dynamic form in the sample application.
+
+From the `app.component.html` file:
+
+```HTML
+<header>
+  <h1>Angular OIE Example App</h1>
+</header>
+
+<main>
+  <app-menu [isAuthenticated]="(isAuthenticated$ | async) ?? false" (startFlow)="onFlow($event)" (logout)="onLogout()"></app-menu>
+  <ng-container *ngIf="idxMessages$ | async as messages">
+    <div>
+      <p *ngFor="let message of messages">{{message.message}}</p>
+    </div>
+  </ng-container>
+  <ng-container *ngIf="nextStep$ | async as nextStep">
+    <app-okta-dynamic-form *ngIf="!(isAuthenticated$ | async)" [nextStep]="nextStep" (userInput)="onInput($event)"></app-okta-dynamic-form>
+  </ng-container>
+  <ng-container *ngIf="isAuthenticated$ | async">
+    <div>
+      <h3>ID Token</h3>
+      <pre>{{idToken$ | async | json}}</pre>
+    </div>
+  </ng-container>
+</main>
+```
+
+From the `app.component.ts` file:
+
+```TypeScript
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss'],
+  providers: [FormTransformerService]
```

Comment:
This LOC (`providers: [FormTransformerService]`) can and should be removed. I accidentally left it in previously but the sample repo has been cleaned up now.

Reply:
Removed.